emulator/blockchain.go

```diff
@@ -1402,31 +1404,57 @@ func (b *Blockchain) commitBlock() (*flowgo.Block, error) {
 		b.vmCtx,
 		fvm.WithBlockHeader(block.ToHeader()),
 	)
+	processTxID := flowgo.MakeID(fmt.Sprintf("process-%d", b.pendingBlock.height))
```

Comment:
This is where i create a new id for the process and systemtx transaction since we cannot store them in the global store with their original id since that is not unique.